docs/modules/ROOT/pages/index.adoc

```diff
@@ -18,11 +18,21 @@ Comprehensive audit trails:: The textual policy language of Cerbos makes it idea
 Cerbos Policy Decision Point (PDP) is built for modern, containerised microservice environments with support for both x86-64 and ARM64 architectures, comprehensive observability integrations (metrics, distributed tracing), REST and gRPC endpoints, and native GitOps support (CI tooling, push-to-deploy). 
 
 
-.Cerbos workflow
+== Cerbos workflow
 * Author Cerbos policies to define access rules for your resources. Optionally, write unit tests for the policies using the Cerbos DSL. 
 * Compile the policies and run tests using the Cerbos CLI.
 * Follow your standard development process to push the changes to production. (E.g. create pull request, run CI tests, get approval and merge to prod branch)
 * Cerbos will automatically pull the latest commits from the production branch and update the policies in place without requiring a restart. Your changes are now rolled out! 
 
+== Authorization as a Service
+It was an explicit design choice to have Cerbos be a standalone service rather than a library for the following reasons:
 
+* By having it as a single service any part of the application stack can call to check permissions regardless of language, deployment or system - Cerbos is totally agnostic in that way.
+* With modern network stacks, the latency is link:https://www.miketheman.net/2021/12/28/container-to-container-communication/[quite negligible] so it isn't an area of concern unless the system is processing millions of requests a second (and even then there are options to support this).
+* Allows the Cerbos core to be written in the most optimal language (Go in this case) rather than constraining options to just languages that can be compiled down to libraries for a handful of common ones.
 
+Cerbos is design in a way that is very aware of the fact that it is in the request path in most cases so a number of key features have been implemented to mitigate this:
+
+* The API is exposed in gRPC to keep overheads low (with an HTTP gateway on top).
+* A sidecar deployment is recommended so that each service instance has its own Cerbos PDP to keep latency as low as possible and even support the use of UNIX sockets.
+* Cerbo is advocating a modern cloud-native approach to dealing with common infrastructure services such as authorization. This is a proven method - link:https://docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/dapr-at-20000-feet[Microsoft Dapr] is a good example at scale.
```

Comment:
```suggestion
The Cerbos approach is a proven, modern, cloud native pattern for delivering language-agnostic infrastructure services. link:https://dapr.io[Microsoft Dapr], link:https://istio.io[Istio] and link:https://linkerd.io[Linkerd] are good examples of popular products utilising similar language-agnostic service APIs to augment applications.
    
Because Cerbos is in the critical request path and expected to handle large volumes of requests, we are obsessive about making Cerbos as fast and as efficient as possible with every release. Cerbos exposes an efficient, low latency gRPC API and is designed to be stateless and lightweight so that it can be deployed as a sidecar right next to  your application. It can even be accessed over Unix domain sockets for extra security and reduced overhead.   
```